src/app/_services/series.service.ts

```diff
@@ -90,8 +91,8 @@ export class SeriesService {
     return this.httpClient.post<void>(this.baseUrl + 'reader/mark-unread', {seriesId});
   }
 
-  getRecentlyAdded(libraryId: number = 0) {
-    return this.httpClient.get<Series[]>(this.baseUrl + 'series/recently-added?libraryId=' + libraryId).pipe(map(series => {
+  getRecentlyAdded(libraryId: number = 0, limit: number) {
```

Comment:
Like I mentioned in your other PR, let's re-use what we already have. So instead of adding limit, let's keep the consistency where we have pageNum?: number, itemsPerPage?: number and use this._addPaginationIfExists(params, pageNum, itemsPerPage).

We just need to change the signature to return PaginatedList<Series> instead and tweak library.component.ts